src/org/zaproxy/zap/extension/ascanrulesBeta/Csrftokenscan.java

```diff
@@ -19,11 +19,7 @@
 
 import java.io.IOException;
 import java.io.StringReader;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.TreeSet;
+import java.util.*;

```

Comment:
Please keep the specific imports vs wildcards.

Reply:
oops, my IDE screw these specific imports, I'll update